configure.ac

```diff
@@ -74,8 +72,14 @@ AC_CHECK_LIB(dl, dlopen, [],
 dnl Checks for libraries
 AC_CHECK_LIB(pthread, pthread_mutex_init, [],
     AC_MSG_ERROR([pthread_mutex_init() not found.  libfabric requires libpthread.]))
-AC_CHECK_LIB(rt, clock_gettime, [],
-    AC_MSG_ERROR([clock_gettime() not found.  libfabric requires librt.]))
+
+AC_SEARCH_LIBS([clock_gettime],[rt posix4],
+		[AC_DEFINE([HAVE_CLOCK_GETTIME], [1],
+			[Define to 1 if have clock_gettime.])],
+		[AS_IF([test -z $(expr "$host_os" : '.*\(darwin\)')],
+			[AC_MSG_ERROR([clock_gettime not found.])],
```

Comment:
Heh -- our comments are getting a little jumbled.  For the always-define-to-0-or-1 stuff, you can do it several ways.  Here's one:

```sh
have_clock_gettime=0
AC_SEARCH_LIBS([clock_gettime],[rt posix4],
		[have_clock_gettime=1],
		[...handle the do-not-have-it-case...])
AC_DEFINE_UNQUOTED(HAVE_CLOCK_GETTIME, [$have_clock_gettime])
```